### ISSUE TITLE
Linux repos use deprecated commands

### ISSUE THREAD

**Issue by jnweiger** [owncloud/client#9180]
Seen in Ubuntu 21.04

The usual command to install the gpg key as advertised in our linux download repo setup is
```
wget -nv http://195.201.128.253/client/btr-2.9.2-rc1.5681-linux/Ubuntu_21.04/Release.key -O - | sudo apt-key add -
```
This still works in Ubuntu 21.04, but triggers  deprecated warning.

Suggest to rewrite the command as 
```
wget -nv http://client-linux-install.jw-qa.owncloud.works/client/btr-2.9.2-rc1.5681-linux/Ubuntu_21.04/Release.key -O - | gpg --dearmor | sudo dd of=/etc/apt/trusted.gpg.d/btr-client-292.gpg
```
(similar ugliness to what opensuse OBS does, maybe a bit less ugly)

**Comment by jnweiger** [comment]
Example: https://github.com/CollaboraOnline/online/issues/1934

**Comment by fmoc** [comment]
Fixed for 3.0+. See https://download.owncloud.com/desktop/ownCloud/daily/3.0/linux/download/.

**Comment by individual-it** [comment]
tested on Ubuntu 22.04 and command work without deprecated warnings
